lib/game/entityHistory.go

```diff
@@ -89,5 +88,11 @@ func (ph *HistoryBuffer) PositionAt(t float64) coords.World {
 	}
 
 	alpha := (t - older.t) / (newer.t - older.t)
-	return older.pos.Lerp(newer.pos, alpha)
+	return &physics.Body{
+		Pos:          *older.body.Pos.Lerp(&newer.body.Pos, alpha),
+		Vel:          *older.body.Vel.Lerp(&newer.body.Vel, alpha),
+		Dir:          *older.body.Dir.Lerp(&newer.body.Dir, alpha),
+		HalfExtents:  newer.body.HalfExtents,
+		CenterOffset: newer.body.CenterOffset,
+	}
```

Comment:
Well this makes it explicit which properties we want to linearly interpolate for this specific use case. There could be cases where we want to linearly interpolate the half extends, like resizing an object.